From 83fb3aa4b241d27d37bbfc55649e3c47e558e08a Mon Sep 17 00:00:00 2001
From: John Mazzitelli <mazz@redhat.com>
Date: Wed, 19 Jan 2022 17:06:32 -0500
Subject: bump kiali to v1.45 (#36900)

* bump kiali to v1.45

* set the image_version in the gen.sh script so it is easier to remember for people to change. now you just need to update the gen.sh script not the values at the same time.
---
 manifests/addons/gen.sh            |  3 +-
 manifests/addons/values-kiali.yaml |  1 -
 samples/addons/kiali.yaml          | 68 +++++++++++++++---------------
 3 files changed, 36 insertions(+), 36 deletions(-)

diff --git a/manifests/addons/gen.sh b/manifests/addons/gen.sh
index 5e103e97a9..640ce70c6c 100755
--- a/manifests/addons/gen.sh
+++ b/manifests/addons/gen.sh
@@ -30,7 +30,8 @@ TMP=$(mktemp -d)
 {
 helm3 template kiali-server \
   --namespace istio-system \
-  --version 1.44.0 \
+  --version 1.45.0 \
+  --set deployment.image_version=v1.45 \
   --include-crds \
   --set nameOverride=kiali \
   --set fullnameOverride=kiali \
diff --git a/manifests/addons/values-kiali.yaml b/manifests/addons/values-kiali.yaml
index 587630cbb0..418ff3dd26 100644
--- a/manifests/addons/values-kiali.yaml
+++ b/manifests/addons/values-kiali.yaml
@@ -6,7 +6,6 @@ deployment:
     sidecar.istio.io/inject: "false"
   accessible_namespaces:
   - '**'
-  image_version: v1.42
   ingress_enabled: false
 login_token:
   signing_key: CHANGEME
diff --git a/samples/addons/kiali.yaml b/samples/addons/kiali.yaml
index ac906a6fce..a43a4f40a3 100644
--- a/samples/addons/kiali.yaml
+++ b/samples/addons/kiali.yaml
@@ -6,12 +6,12 @@ metadata:
   name: kiali
   namespace: istio-system
   labels:
-    helm.sh/chart: kiali-server-1.44.0
+    helm.sh/chart: kiali-server-1.45.0
     app: kiali
     app.kubernetes.io/name: kiali
     app.kubernetes.io/instance: kiali
-    version: "v1.44.0"
-    app.kubernetes.io/version: "v1.44.0"
+    version: "v1.45.0"
+    app.kubernetes.io/version: "v1.45.0"
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/part-of: "kiali"
 ...
@@ -23,12 +23,12 @@ metadata:
   name: kiali
   namespace: istio-system
   labels:
-    helm.sh/chart: kiali-server-1.44.0
+    helm.sh/chart: kiali-server-1.45.0
     app: kiali
     app.kubernetes.io/name: kiali
     app.kubernetes.io/instance: kiali
-    version: "v1.44.0"
-    app.kubernetes.io/version: "v1.44.0"
+    version: "v1.45.0"
+    app.kubernetes.io/version: "v1.45.0"
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/part-of: "kiali"
 data:
@@ -55,7 +55,7 @@ data:
       image_name: quay.io/kiali/kiali
       image_pull_policy: Always
       image_pull_secrets: []
-      image_version: v1.42
+      image_version: v1.45
       ingress:
         additional_labels: {}
         class_name: nginx
@@ -85,7 +85,7 @@ data:
       service_annotations: {}
       service_type: ""
       tolerations: []
-      version_label: v1.44.0
+      version_label: v1.45.0
       view_only_mode: false
     external_services:
       custom_dashboards:
@@ -119,12 +119,12 @@ kind: ClusterRole
 metadata:
   name: kiali-viewer
   labels:
-    helm.sh/chart: kiali-server-1.44.0
+    helm.sh/chart: kiali-server-1.45.0
     app: kiali
     app.kubernetes.io/name: kiali
     app.kubernetes.io/instance: kiali
-    version: "v1.44.0"
-    app.kubernetes.io/version: "v1.44.0"
+    version: "v1.45.0"
+    app.kubernetes.io/version: "v1.45.0"
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/part-of: "kiali"
 rules:
@@ -216,12 +216,12 @@ kind: ClusterRole
 metadata:
   name: kiali
   labels:
-    helm.sh/chart: kiali-server-1.44.0
+    helm.sh/chart: kiali-server-1.45.0
     app: kiali
     app.kubernetes.io/name: kiali
     app.kubernetes.io/instance: kiali
-    version: "v1.44.0"
-    app.kubernetes.io/version: "v1.44.0"
+    version: "v1.45.0"
+    app.kubernetes.io/version: "v1.45.0"
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/part-of: "kiali"
 rules:
@@ -323,12 +323,12 @@ kind: ClusterRoleBinding
 metadata:
   name: kiali
   labels:
-    helm.sh/chart: kiali-server-1.44.0
+    helm.sh/chart: kiali-server-1.45.0
     app: kiali
     app.kubernetes.io/name: kiali
     app.kubernetes.io/instance: kiali
-    version: "v1.44.0"
-    app.kubernetes.io/version: "v1.44.0"
+    version: "v1.45.0"
+    app.kubernetes.io/version: "v1.45.0"
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/part-of: "kiali"
 roleRef:
@@ -348,12 +348,12 @@ metadata:
   name: kiali-controlplane
   namespace: istio-system
   labels:
-    helm.sh/chart: kiali-server-1.44.0
+    helm.sh/chart: kiali-server-1.45.0
     app: kiali
     app.kubernetes.io/name: kiali
     app.kubernetes.io/instance: kiali
-    version: "v1.44.0"
-    app.kubernetes.io/version: "v1.44.0"
+    version: "v1.45.0"
+    app.kubernetes.io/version: "v1.45.0"
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/part-of: "kiali"
 rules:
@@ -381,12 +381,12 @@ metadata:
   name: kiali-controlplane
   namespace: istio-system
   labels:
-    helm.sh/chart: kiali-server-1.44.0
+    helm.sh/chart: kiali-server-1.45.0
     app: kiali
     app.kubernetes.io/name: kiali
     app.kubernetes.io/instance: kiali
-    version: "v1.44.0"
-    app.kubernetes.io/version: "v1.44.0"
+    version: "v1.45.0"
+    app.kubernetes.io/version: "v1.45.0"
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/part-of: "kiali"
 roleRef:
@@ -406,12 +406,12 @@ metadata:
   name: kiali
   namespace: istio-system
   labels:
-    helm.sh/chart: kiali-server-1.44.0
+    helm.sh/chart: kiali-server-1.45.0
     app: kiali
     app.kubernetes.io/name: kiali
     app.kubernetes.io/instance: kiali
-    version: "v1.44.0"
-    app.kubernetes.io/version: "v1.44.0"
+    version: "v1.45.0"
+    app.kubernetes.io/version: "v1.45.0"
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/part-of: "kiali"
   annotations:
@@ -435,12 +435,12 @@ metadata:
   name: kiali
   namespace: istio-system
   labels:
-    helm.sh/chart: kiali-server-1.44.0
+    helm.sh/chart: kiali-server-1.45.0
     app: kiali
     app.kubernetes.io/name: kiali
     app.kubernetes.io/instance: kiali
-    version: "v1.44.0"
-    app.kubernetes.io/version: "v1.44.0"
+    version: "v1.45.0"
+    app.kubernetes.io/version: "v1.45.0"
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/part-of: "kiali"
 spec:
@@ -458,24 +458,24 @@ spec:
     metadata:
       name: kiali
       labels:
-        helm.sh/chart: kiali-server-1.44.0
+        helm.sh/chart: kiali-server-1.45.0
         app: kiali
         app.kubernetes.io/name: kiali
         app.kubernetes.io/instance: kiali
-        version: "v1.44.0"
-        app.kubernetes.io/version: "v1.44.0"
+        version: "v1.45.0"
+        app.kubernetes.io/version: "v1.45.0"
         app.kubernetes.io/managed-by: Helm
         app.kubernetes.io/part-of: "kiali"
         sidecar.istio.io/inject: "false"
       annotations:
-        checksum/config: c3772da766b94abe8269b6eaed2ecb03d1c1b6d5243f3b21f892de4590f2e360
+        checksum/config: 0b7a16e51830df3ef2ff816eb3d43341a23c822b38fea95050b3707ba2ca119e
         prometheus.io/scrape: "true"
         prometheus.io/port: "9090"
         kiali.io/dashboards: go,kiali
     spec:
       serviceAccountName: kiali
       containers:
-      - image: "quay.io/kiali/kiali:v1.42"
+      - image: "quay.io/kiali/kiali:v1.45"
         imagePullPolicy: Always
         name: kiali
         command:
-- 
2.35.3

